JavadocStyleGuide.md

```diff
@@ -0,0 +1,223 @@
+- The first line that begins with an `@` character ends the description; you cannot continue the description following block tags.
+- Block tags must be added in order.
+- The last line contains the end-comment delimiter ( `*/`).
+
+So lines won't wrap, limit any doc-comment lines to 80 characters.
+
+### 1.1.2 HTML
+
+Javadoc permits only a subset of HTML tags: `"a", "abbr", "acronym", "address", "area", "b", "bdo", "big", "blockquote", "br", "caption", "cite", "code", "colgroup", "dd", "del", "dfn", "div", "dl", "dt", "em", "fieldset", "font", "h1", "h2", "h3", "h4", "h5", "h6", "hr", "i", "img", "ins", "kbd", "li", "ol", "p", "pre", "q", "samp", "small", "span", "strong", "sub", "sup", "table", "tbody", "td", "tfoot", "th", "thead", "tr", "tt", "u", "ul", "var"`.
+
+Attempt to limit your usage of HTML to: `"h1", "h2", "h3", "h4", "em", "li", "ol", "p", "ul"`.
+
+Some HTML element types allow authors to omit end tags: `<p>` and `<li>`.
+
+### 1.1.3 Paragraphs
+
+One blank line—that is, a line containing only the aligned leading asterisk (`*`)—appears between the last paragraph and before the group of block tags if present:
+
+```java
+/**
+ * that draw the image will incrementally paint on the screen.
+ *
+ * @param  url  an absolute URL giving the base location of the image
+ */
+```
+
+Each paragraph except the first has `<p>` immediately before the first word, with no space after it:
+
+```java
+/**
+ * argument is a specifier that is relative to the url argument.
+ * <p>This method always returns immediately, whether or not the
+ */
+```
+
+Alternatively, a `<p>` can be placed on a separate line:
+
+```java
+/**
+ * argument is a specifier that is relative to the url argument.
+ * <p>
+ * This method always returns immediately, whether or not the
+ */
+```
+
+The inline tag `{@link URL}`, which converts to an HTML hyperlink pointing to the documentation for the URL class. 
+This inline tag can be used anywhere that a comment can be written, such as in the text following block tags:
+
+```java
+/**
+ * The url argument must specify an absolute <a href="#{@link}">{@link URL}</a>. The name
+ */
+```
+
+HTML tags for other block-level elements, such as `<ul>` or `<table>`, are not preceded with `<p>`.
+
+### 1.1.4 Symbols
+
+Entities for the less than symbol (<) and the greater than symbol (>) should be written as `&lt`; and `&gt;`. 
+Similarly, the ampersand (&) should be written as `&amp;`.
+
+```java
+/**
+ * The url argument must specify an absolute <a href="#{@link}">{@link URL}</a>. The name
+ */
+```
+### 1.1.5 Links
+
+The `{@link <class or method reference>}` tag is specifically used to link to the Javadoc of other classes and methods. 
+This is an inline tag that converts to an HTML hyperlink pointing to the documentation of the given class or method reference
+
+```java
+/**
+ * This &amp; that.
+ */
+```
+
+### 1.1.6 Code keywords, names, and variables
+
+Use `<code>...</code>` style for keywords and names including:
```

Comment:
I much prefer `{@code ... }` for two main reasons:

1. It is more concise.
2. HTML characters are automatically-escaped. 

For example, I can write `{@code List<String>}` instead of `<code>List&lt;String&gt;</code>`

Reply:
I think regardless of our decision both options should be mentioned in the style guide. For instance, we should explicitly say that using `{@code ...}` is preferred over the HTML equivalent or that it does not matter which one is used.

Reply:
Both are ok. Updated the style guide accordingly.